audits/manifest/background-color.js

```diff
@@ -26,13 +26,16 @@ class ManifestBackgroundColor {
     return 'Contains background_color';
   }
 
-  static audit(inputs) {
-    let hasBackgroundColor = false;
-    const manifest = inputs.manifest;
+  static hasBackgroundColorValue(manifest) {
+    /* eslint-disable camelcase */
+    return Boolean(manifest !== undefined &&
```

Comment:
Also I duno when we would use || it is just an example of why a cast is important for these annd or OR chains of fun